tests/Http/Three/PostTest.php

```diff
@@ -372,44 +372,6 @@ public function testValidationUpdatingAPost()
         $this->assertEquals('The caption may not be greater than 140 characters.', $json['errors']['caption'][0]);
     }
 
-    /**
-     * Test that non-authenticated user's/apps can't update posts.
-     *
-     * @return void
-     */
-    public function testUnauthenticatedUserUpdatingAPost()
```

Comment:
Can we keep this test?

Reply:
ah yes, thanks for pointing this out! I misread. I had to update with the new error message we're getting now below. But I'm not exactly sure why this is happening now with a different error message?

Reply:
That's because this is using the new [custom authorization logic](https://github.com/DoSomething/rogue/blob/a3452e43130634c1076a637911c209361257d301/app/Http/Controllers/Three/PostsController.php#L162) in the controller, rather than [Gateway's `role` middleware](https://github.com/DoSomething/gateway/blob/d1479758bb276e8ddb950d49cc3827322684a819/src/Server/Middleware/RequireRole.php#L31). It might make sense to update the messaging to make it more clear that it's an issue with the user's permissions on _that_ particular post, but otherwise 👍 from me!

Reply:
oh duh! Thanks so much for explaining, @DFurnes ! Updated the message to be more explicit :)